src/components/SelectionList/BaseSelectionList.js

```diff
@@ -156,9 +177,36 @@ function BaseSelectionListRadio(props) {
         };
     };
 
+    const renderSectionHeader = ({section}) => {
+        if (!section.title || _.isEmpty(section.data)) {
+            return null;
+        }
+
+        return (
+            // Note: The `optionsListSectionHeader` style provides an explicit height to section headers.
+            // We do this so that we can reference the height in `getItemLayout` –
+            // we need to know the heights of all list items up-front in order to synchronously compute the layout of any given list item.
+            // So be aware that if you adjust the content of the section header (for example, change the font size), you may need to adjust this explicit height as well.
```

Comment:
Should we use `/** */` (_multiline comments_) everywhere?

Reply:
Is this rule in the guidelines? I don't recall seeing it. Generally, I prefer multiline comments for function signatures, and I find single-line comments better for readability when commenting inside a code block

Reply:
Anyway, this was copied over from the original implementation I'm refactoring

Reply:
I read it [here](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#method-naming-and-code-documentation): _"Add descriptions to all propTypes using a **block comment** above the definition."_. 
However, it is not a topic to block your PR. It is up to you 🙂

Reply:
Cool, thanks for the link. But I'll leave like this here because it's not propTypes. Thanks!